src/events/cd-event-details.vue

```diff
@@ -127,6 +152,12 @@
       flex: 4;
       max-width: 340px;
 
+      &--dojo {
+        margin: 24px 0;
+      }
+      &--time {
+        margin: 24px 0 48px 0;
```

Comment:
Why the difference in margin between this and `&--dojo`?

Reply:
Because 48px is huge. Also because it is not the same kind of element. "hosted-by" is a pre- of all left-column items with a border-bottom. See ticket screenshot for more :) https://trello.com/c/wMuZBelr